src/NHibernate/Criterion/ICriteriaQuery.cs

```diff
@@ -6,6 +6,15 @@
 
 namespace NHibernate.Criterion
 {
+	//TODO 6.0: Add to ICriteriaQuery interface
+	internal interface ICriteriaQueryNextVer
```

Comment:
nit: I don't like the name

Reply:
It's internal and name explicitly suggests that it will be merged to main interface. What's not to like? )  I like it! Your suggestion?

Reply:
I got rid of interface

Reply:
Well, I thought also about "niting" here, but have let it due to being internal.

Reply:
So what's wrong with 'NextVer' suffix? I just want to stop thinking how to name my next temp internal interface that should be merged. I thought it's a good plan to use some name convention for such interfaces. If  `ISomethingNextVer` is not good enough can you suggest something better?

Reply:
I intentionally prefixed my comment with [‘nit:’](https://stackoverflow.com/q/27810522)

Reply:
I understand the rationale here, and I do not have a better idea, that is was I had not written anything initially. I would not oppose this practice, I was just sharing a feeling.